maven/src/main/java/org/commonjava/maven/galley/maven/model/view/MavenPomView.java

```diff
@@ -1034,30 +1038,30 @@ protected boolean containsExpression( final String value )
      * @param activeProfileIds The profiles to consider active, whose elements should be treated as merged with
      *   the main POM body
      */
+
     public String resolveExpressions( final String value, final String... activeProfileIds )
     {
         if ( !containsExpression( value ) )
         {
-            //            logger.info( "No expressions in: '{}'", value );
             return value;
         }
 
-        synchronized ( this )
-        {
-            if ( ssi == null )
-            {
-                ssi = new StringSearchInterpolator();
-            }
-            ssi.addValueSource( new MavenPomViewVS( this, activeProfileIds ) );
-        }
+        List<String> prefixs = new ArrayList<>();
```

Comment:
Since you have to do the null check in `resolveExpressions(String, RecursionInterceptor, String...)` anyway, you could probably move this down to that method, and call with a null `ri` here...